src/components/Login/Login.jsx

```diff
@@ -76,7 +76,7 @@ export class Login extends React.Component {
                     </div>
                     <Monster />
                     <form className={styles.loginForm} onSubmit={this.sendLogin.bind(this)}>
-                        <h3 data-cy="login-error" className={styles.errorText}>{}</h3>
+                        <h3 data-cy="login-error" className={styles.errorText}>{(typeof this.props.error === 'string') ? this.props.error : ""}</h3>
```

Comment:
Does this need to be a ternary?

Reply:
There seems to be a point where I'm setting this error to be the actual error we receive from the HTTP request, which is not a string, so React gives an error. I was thinking that when we do the error handling ticket, this can all be fixed

Reply:
Ok, I think that answers my question as well, it would be good to remove the type checking at some point but if that needs to be in the future then that's fine